inst/JOSS/paper.Rmd

```diff
@@ -187,18 +193,33 @@ ha_record <- bcdc_get_record("7bc6018f-bb4f-4e5d-845e-c529e3d1ac3b")
 ## Have a quick look at the geospatial columns to help with filter or select
 bcdc_describe_feature(ha_record)
 
+## Naively download the whole data object, then filter it
+system.time(
+  all_ha <- bcdc_get_data(ha_record, resource = "93b79a3c-2da4-4fd4-b953-2f5c690db430")
```

Comment:
I think we should consider changing our example away for Health Authority. It is now a layer in bcmaps and is an easier way to get it. Highlighting it here _might_ be confusing.

Reply:
Sure - so are most of the other largish admin polygon layers that would demonstrate the benefit! Any suggestions?

Reply:
Changed to Species at Risk occurrences - VI Marmots!